FAQ Page Updates

TimCliff opened the pull request:
This pull request updates the FAQ page with additional content. The content has been copy-edited by @pfunks, and reviewed by Mitchell.

The PR also makes minor updates to the welcome page, so that the TOC format and CSS are the same between the two pages.

TimCliff commented:
link to steemit announcement/discussion post:
https://steemit.com/faq/@timcliff/major-updates-to-the-steemit-com-faq-page-pull-request-submitted

sneak commented on app/help/en/faq.md:
```diff
@@ -3,58 +3,1103 @@
+### Community
+- <a href="/faq.html#Is_there_an_Etiquette_Guide_for_Steemit">Is there an Etiquette Guide for Steemit?</a>
+- <a href="/faq.html#Am_I_required_to_verify_my_identity">Am I required to verify my identity?</a>
+- <a href="/faq.html#What_are_whales__dolphins_and_minnows">What are whales, dolphins and minnows?</a>
```
usage of the terms "whales", "dolphins", and "minnows" are not supported by steemit and we would like to avoid these terms in all official communications.

sneak commented on app/help/en/faq.md:
```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
+## How does Steemit, Inc. earn money?
 
-Steem is the name of both the database and digital points, which are a form of cryptocurrency that Steemit.com plugs into. 
+Over time, Steemit will allow advertisers and bloggers to promote content by buying and burning SBD/STEEM. Steemit can benefit from sales of Steem to advertisers in the cryptocurrency markets or offer advanced services to these advertisers and bloggers.
```
we do not have any plans at this time to allow advertising on the site, and may never do so.  please remove the term "advertisers" from the FAQ.

sneak commented on app/help/en/faq.md:
```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
+Steemit, Inc. is a privately held company and is not available for public investment.
 
-You can earn digital points on Steemit by:
+Though not considered an investment, you can speculate on the platform's success by purchasing STEEM tokens. You can power up these tokens into STEEM Power, which grants more influence in the Steem platform.
```
Please remove the term "speculate".

sneak commented on app/help/en/faq.md:
```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
+Currently, the Steem and Steemit logos are the same and is free to use. In the future, Steemit, Inc. will have its own logo so that it can be distinguished from Steem. The Steemit logo will be proprietary while Steem and its three S-shaped squiggles will remain open for public use.
 
-Steemit is not a scam. A scam begins with you having to give money to someone else. Steemit is not a middleman to any of the digital points allocations users receive. Steemit doesn't ask you to invest anything.  The Steem blockchain and points system is entirely its own entity, similar to Bitcoin, and it should be treated with a high level of due diligence for anyone looking to leverage its digit ...
+<a href="/faq.html#Table_of_Contents_Steemit__Inc">&#11025;</a>
```
Please add a note suggesting that people avoid use of the term "Steemit" in their own products and instead refer to the Steem Blockchain or Steem Platform.  Steemit means steemit.com.

sneak commented on app/help/en/faq.md:
```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
+## Did Steemit "pre-mine" tokens?
 
-The founders of Steemit currently own about half of the Steem supply. They are publicly committed to reducing their percentage ownership by distributing Steemit’s digital points as means for covering costs of scaling, operations and signing new users up the platform. 
+The challenge for a blockchain startup company is that they need tokens to sell to get funding for the development. If the blockchain is launched so that anybody can mine as much as they want, founders will get very little and may not be able to fund the development.
```
please remove the entirety of this answer except the last paragraph on line 1167.

sneak commented on app/help/en/faq.md:
```diff
@@ -64,182 +1109,261 @@ https://www.linkedin.com/in/nedscott
+**Active key** - The active key is meant for more sensitive tasks such as transferring funds, power up/down transactions, converting Steem Dollars, voting for witnesses, updating profile details and avatar, and placing a market order. 
 
-No, you are not being paid by Steemit. The Steem network pays you. It is your responsibility to determine what, if any, taxes apply to the transactions you make, and it is your responsibility to report and remit the correct tax to the appropriate tax authority. By creating an account, you agree that Steemit is not responsible for determining whether taxes apply to your Steem transactions or for c ...
+**Memo key** - The memo key is the only key that can decrypt private messages or memos sent to your account. Currently these features are not present on Steemit.com
```
Please amend this answer to read "Currently the memo key is not used."

TimCliff commented:
Hi @sneak - I made the requested changes:
- Removed all references to whales, dolphins and minnows.
- Removed question/answer pertaining to Steemit allowing advertisers
- Reworded investing question to remove the word 'speculate': Though not considered an investment, you can purchase STEEM tokens which can go up or down in value.
- Removed requested language from the "pre-mine" section.
- Updated Memo key to "Currently the memo key is not used."

Note, I also made a separate commit to change the 'up arrow' links back to the TOC in both the FAQ and welcome page to use the ^ character, since @pfunks reported that the 'special character' I was using was not showing up in certain browsers.

valzav commented:
@sneak could you update your "requested changes" review, so we can merge if it's good.

sneak commented:
thanks @TimCliff!  this is great and super needed work!  (apologies for my lag in merging it.)

TimCliff commented:
Thanks @sneak! No worries, I know you all are busy 😀

I'll try to check in an update soon with a question about the new signup process.
